core/jobs/batch_jobs/exp_migration_jobs.py

```diff
@@ -233,6 +234,17 @@ def _update_exploration(
                     opportunity_services
                     .update_opportunity_with_updated_exploration(
                         migrated_exp.id))
+            exp_versions_diff = (
+                exp_domain.ExplorationVersionsDiff(list(exp_changes))
+            )
+            # Trigger statistics model update.
+            new_exp_stats = stats_services.get_stats_for_new_exp_version(
+                migrated_exp.id, migrated_exp.version,
+                list(migrated_exp.states.keys()),
+                exp_versions_diff, None)
```

Comment:
Can we force the `None` to be passed as a keyword argument (for better readability).

Reply:
Done.